ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml

```diff
@@ -5204,14 +5204,12 @@ periodics:
         secretName: build-farm-credentials
 - agent: kubernetes
   cluster: api.ci
+  cron: '@yearly'
   decorate: true
-  interval: 2h
   labels:
     job-env: aws
     job-release: "4.7"
     job-test: e2e
-    pj-rehearse.openshift.io/can-be-rehearsed: "true"
```

Comment:
Can you add this back so the job can be rehearsed? I believe it should work with the `releases` section that was added below.